packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -465,6 +466,126 @@ class CommentsRepo extends AbstractRepo<"Comments"> {
       WHERE "forumEventId" = $1 AND "userId" = $3
     `, [forumEventId, latestVote, userId]);
   }
+
+  /**
+   * Get comments for the UltraFeed
+   */
+  async getCommentsForFeed(
+    context: ResolverContext, 
+    maxTotalComments = 1000,
+    lookbackWindow = '90 days'
+  ): Promise<FeedCommentFromDb[]> {
+    const db = this.getRawDb();
+    const userId = context.userId;
+    const initialCandidateLimit = 100; // Limit for the first pass
+
+    const UNIVERSAL_COMMENT_FILTER_CLAUSE = `
+      c.deleted IS NOT TRUE
+      AND c.retracted IS NOT TRUE
+      AND c."authorIsUnreviewed" IS NOT TRUE
```

Comment:
I think `authorIsUnreviewed` is already part of `getViewableCommentsSelector`.  (And, to simplify your life, you might want to concatenate that selector with `UNIVERSAL_COMMENT_FILTER_CLAUSE` before using it, so you don't need to include both things in every location where you want them.)